aten/src/ATen/native/cpu/FlashAttentionKernel.cpp

```diff
@@ -237,6 +269,8 @@ void cpu_flash_attention(
       : 0;
   int64_t mStrideM =
       has_attn_mask ? attn_mask.value().stride(2) : 0;
+  int64_t mStrideN =
+      has_attn_mask ? attn_mask.value().stride(3) : 0;
```

Comment:
Why don't we also check the size here as `mStrideB` and `mStrideH` do? The stride could be of any value when size is 1.

Reply:
Thanks, added!